STEERING-COMMITTEE.md

```diff
@@ -60,28 +59,27 @@ committee currently recognizes this delegated authority for:
 
 KSC meets every two weeks, or as-needed. Meetings are held online.
 
-Given the private nature of many of these discussions (e.g. privacy, private
+Given the private nature of some of these discussions (e.g. privacy, private
 emails to the committee, code of conduct violations, escalations, disputes
-between members, security reports, etc.) meetings are held in private.
+between members, security reports, etc.) some meetings are held in private.
 
-Meeting notes are available to members of the knative-dev mailing list (link to
-be added).
+Meeting notes are available to members of the [knative-dev mailing
+list](https://groups.google.com/forum/#!forum/knative-dev). Public meetings will
+be recorded and the recordings made available publicly.
 
 Questions and proposals for changes to governance are posted as
 [issues in the docs repo](https://github.com/knative/docs/issues), and the KSC
-invites your feedback there. See [Getting in touch](#getting-in-touch) for other
+invites your feedback there. See [Raising Issues](#raising-issue-to-the-steering-committee) for other
 options.
 
 ## Committee members
 
-Seats on the Steering Committee are held by an organization, not by the
-individual.
+Seats on the Steering Committee are held by an individual, not by their employer.
```

Comment:
This is shortest line on this PR and the strongest statement of open source values. 💯